src/containers/RegistrationContainer.js

```diff
@@ -144,10 +153,21 @@ const RegistrationContainer = props => {
     }
   }
 
-  const profileCreationAction = () => {
+  const getUsername = () => {
```

Comment:
This could be in utils since it is used in at least two files.

Reply:
Moved to utils.